src/lib/archiving-api.js

```diff
@@ -21,6 +21,8 @@ const archivingApi = {
     return Promise.resolve('{ "success": true }');
   },
 
+  ping: async () => { },
```

Comment:
Continuing from above: 
```suggestion
  available: async () => true,
```

This way you don't have an empty function either, which makes it easier to understand.